src/documentation/gtfs_to_ntfs_specs.md

```diff
@@ -1,12 +1,14 @@
 # GTFS reading specification
 
 ## Purpose
-This document aims to describe how the [GTFS format](https://developers.google.com/transit/gtfs/reference) is read in the Navitia Transit Model. To improve readability of this document, the specification will describe the transformation of a GTFS feed into a [NTFS feed](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md) (which is a bunch of csv files accordingly to the memory Navitia Transit Model).
+This document aims to describe how the [GTFS] format is read in the Navitia Transit Model. To improve readability of this document, the specification will describe the transformation of a GTFS feed into a [NTFS feed](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md) (which is a bunch of csv files accordingly to the memory Navitia Transit Model).
```

Comment:
How about doing the same for NTFS?
```suggestion
This document aims to describe how the [GTFS] format is read in the Navitia Transit Model. To improve readability of this document, the specification will describe the transformation of a GTFS feed into a [NTFS] feed(which is a bunch of csv files accordingly to the memory Navitia Transit Model).

[NTFS]: https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md
```

Reply:
I thought about this too (and there are a lot of "little" tasks like that I identified and stashed).
But I would rather do it in a separate PR to avoid messing with understanding the main purpose of this one.
I propose to let this comment unresolved here but address it in a following PR.

Reply:
Done in https://github.com/CanalTP/transit_model/pull/657